prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -533,6 +543,11 @@ class RESTTransport(private val httpClient: HttpClient? = null) : ITransport {
                             // create JSON object for the BODY. This encodes "/" character as "//", needed for WA to accept as valid JSON
                             JSONObject().put("body", message.toString(Charsets.UTF_8)).toString()
                         }
+                        "application/hl7-v2" -> {
+                            // The following line doesn't work. It shows one seg on their server
+                            val filteredMsg = message.toString(Charsets.UTF_8).replace("\n", "\r").dropLast(1) + "\r"
```

Comment:
Do we know if the HAPI FHIR library is currently including linefeeds instead of carriage returns?

(I ran into the same issue in sending HL7 to Oracle RLN via Postman; I was hoping the conversion via HAPI would not use LF as [I think] it violates the HL7 spec.)

Reply:
Yes, the HAPI FHIR lib is including linefeed '/n' instead.